Sources/StreamVideoUIKit/CallViewController.swift

```diff
@@ -14,9 +14,9 @@ open class CallViewController: UIViewController {
     
     private var cancellables = Set<AnyCancellable>()
     
-    public static func make(with viewModel: CallViewModel? = nil) -> CallViewController {
```

Comment:
Why is it mandatory to provide the view model now?

Reply:
I brought this one back to not break customers integration but i basically would like to deprecate it for a few reasons:
1. There is no reason to have optionality for the viewModel as internally we always ensure that there will be one
2. Static methods like this one aren't really swifty and are more difficult to be discovered (and require mention and explanation in docs). We can rely on convenience initializers to achieve the same result